compiler/rustc_resolve/src/late.rs

```diff
@@ -3040,6 +3064,21 @@ impl<'a: 'ast, 'b, 'ast, 'tcx> LateResolutionVisitor<'a, 'b, 'ast, 'tcx> {
                     },
                 );
             }
+            AssocItemKind::Delegation(box delegation) => {
+                debug!("resolve_implementation AssocItemKind::Delegation");
+                let emitted = self.check_trait_item(
+                    item.id,
+                    item.ident,
+                    &item.kind,
+                    ValueNS,
+                    item.span,
+                    seen_trait_items,
+                    |i, s, c| MethodNotMemberOfTrait(i, s, c),
+                );
+
+                // do not emit same errors twice during path resolution
```

Comment:
They are not the same errors.
One error says that the path is unresolved, and another says that there's no such item in the trait, both may be reported independently.
I wouldn't bother deduplicating them.